src/utils/map.c

```diff
@@ -1,16 +1,25 @@
 #include <cub3D.h>
 
-void	set_map_value(t_map *map, int line, int column, char c)
+void	set_floor_color(t_map *map, int color)
+{
+	map->floor_color = color;
+}
+
+void	set_ceiling_color(t_map *map, int color)
+{
+	map->ceiling_color = color;
+}
+
+void	set_cube_value(t_map *map, int line, int column, char c)
```

Comment:
replace char c with a enum?

Reply:
the way set cube value is used is that it checks if the char c is a valid cube enum value and if so sets it to the corresponding enum value. this is due to the functions use in the parser - until that behaviour gets us in to trouble somehow i would want to keep it that way